omegaconf/basecontainer.py

```diff
@@ -281,6 +281,9 @@ def _map_merge(dest: "BaseContainer", src: "BaseContainer") -> None:
         assert isinstance(src, DictConfig)
         src_type = src._metadata.object_type
 
+        if src._is_interpolation():
+            dest._set_value(src._value())
+            return
```

Comment:
Please explain this one.

Reply:
Merging an interpolation to a container is basically keeping the interpolation in _dest_ therefore, dest's value should be src's value.
If I don't do this check here it will fail in `for key, src_value in src.items_ex(resolve=False):`(line 307). 
To sum up, it's the same as line 289 where `if src._is_missing(): dest._set_value("???") `

Reply:
Ok. See that there is a comment explaining it in 288.
Add a similar comment.